### PR TITLE
[WIP] Catch invalid argument

### PR DESCRIPTION
**Pull request by Auxilus**
`history -z` should show invalid argument error rather than showing the history..

### CODE REVIEW

File: lib/msf/ui/console/command_dispatcher/core.rb [hunk 1]
```diff
@@ -484,7 +484,13 @@ def cmd_history(*args)
     else
       limit = @history_limit
     end
-
+    args.each do |arg|
```
Comment (wvu-r7) [inline lib/msf/ui/console/command_dispatcher/core.rb]: Why are you not using the `parse` below? Just stick an `else` at the end.

Comment (wvu-r7) [reply lib/msf/ui/console/command_dispatcher/core.rb]: Looks like `-n` takes an arg. Nevermind.

**Comment by Auxilus** [comment]
current output for `history -z`
```
msf5 > history -z
[-] Invalid argument passed

Usage: history [options]

Shows the command history.
If -n is not set, only the last 100 commands will be shown.

OPTIONS:

    -a        Show all commands in history.
    -h        Help banner.
    -n <opt>  Show the last n commands.
    -u        Show only unique commands.
msf5 >
```

**Comment by Auxilus** [comment]
I've tested some other commands with these changes, no errors yet..

**Comment by Auxilus** [comment]
`cmd_sessions` also needs to be modified to catch invalid argument.

**Comment by Auxilus** [comment]
`cmd_sessions`,  `cmd_irb`, `cmd_threads` are also modified

**Pull request closed without merging**